doc/source/configuration.rst

```diff
@@ -163,12 +161,18 @@ The Nextflow configuration file is used to configure various settings for the *k
         
       Has to match enabled container type (see above).
 
-    * **withName** defines settings for individual processes in the *ksrates* Nextflow pipeline, for example:
+    * **withName** defines settings for individual processes in the *ksrates* Nextflow pipeline.
+    
+      There are 11 processes in the pipeline, 6 of which (``checkConfig``, ``setupCorrection``, ``setParalogAnalysis``, ``setOrthologAnalysis``,  ``doRateCorrection`` and ``drawTree``) are by default run locally because they execute minimal calculations. The remaining 5 processes (``estimatePeak``, ``plotOrthologDistrib``, ``paralogsAnalyses``, ``wgdParalogs`` and ``wgdOrthologs``) are instead run by default on a cluster, if available, and can be configured under this section of the Nextflow configuration file. ``wgdParalogs`` and ``wgdOrthologs`` are the most computationally demanding and it is advised to assign them a higher computational power than the others (suggested: parallelization with about 10 threads and 2GB memory each).
+    
+      Examples of available settings are (for a complete list see the `Nextflow documentation <https://www.nextflow.io/docs/latest/process.html#process-directives>`__):
     
     	* **clusterOption** any native configuration option accepted by your cluster submit command. You can use it to request non-standard resources or use settings that are specific to your cluster and not supported out of the box by Nextflow.
-    	* **beforeScript** allows you to execute a custom (Bash) snippet before the main process script is run. This may be useful to initialise the underlying cluster environment or for other custom initialisation, for example it can be used to load required dependencies if one of the container is not used, provided that the cluster has those dependencies installed.
-    	
-      For a complete list of available settings, see the `Nextflow documentation <https://www.nextflow.io/docs/latest/process.html#process-directives>`_. The processes ``wgdParalogs`` and ``wgdOrthologs`` are the most computationally demanding and it is advised to assign them a higher computational power than the other processes.
+    	* **beforeScript** allows you to execute a custom (Bash) snippet before the main process script is run. This may be useful to initialise the underlying cluster environment or for other custom initialisation, for example it can be used to load required dependencies if one of the container is not used, provided that the cluster has those dependencies installed. The dependencies per process are the following:
```

Comment:
```suggestion
    	* **beforeScript** allows you to execute a custom (Bash) snippet before the main process script is run. This may be useful to initialise the underlying compute cluster environment or for other custom initialisation, for example it can be used to load required dependencies if one of the *ksrates* containers is not used, provided that the cluster has those dependencies installed. In that case, the required external dependencies (see also the `wgd Documentation  <https://wgd.readthedocs.io/en/latest/index.html#external-software>`__ for the *ksrates* Nextflow processes are:
```

Reply:
Since a parenthesis is missing after the link I've commited this changes plus the parenthesis in a separate commit! I'll close this conversation. Commit: #0df51b9